src/porepy/grids/grid_bucket.py

```diff
@@ -556,6 +578,36 @@ def assign_node_ordering(self):
                 # Assign new value
                 n['node_number'] = counter
                 counter += 1
+#------------------------------------------------------------------------------#
+
+    def update_node_ordering(self, removed_number):
```

Comment:
I have only removed one node at a time so far. But for large fracture sets I agree that it would be useful to be able to do multiple nodes at a time. I'll look into it.